datadreamer/utils/luxonis_dataset_converter.py

```diff
@@ -17,9 +17,14 @@ class LuxonisDatasetConverter(BaseConverter):
     """Class for converting a dataset to LuxonisDataset format."""
 
     def __init__(
-        self, dataset_plugin: str = None, dataset_name: str = None, seed: int = 42
+        self,
+        dataset_plugin: str = None,
+        dataset_name: str = None,
+        seed: int = 42,
+        is_instance_segmentation: bool = False,
```

Comment:
If this is True then let's for now print a warning that instance segmentation will be treated as semantic segmentation until we add support for instance segmentation in luxonis-ml.